runtime/src/gpu/cuda/gpu-cuda.c

```diff
@@ -36,9 +38,14 @@
 #include <assert.h>
 #include <stdbool.h>
 
+// this is compiler-generated
+extern const char* chpl_gpuBinary;
+
 static CUcontext *chpl_gpu_primary_ctx;
+static CUmodule *chpl_gpu_cuda_modules;
```

Comment:
Might seem obvious but I'd add a comment to the right of this indicating: // array indexed by device ID (we load the same module once for each GPU).

The name makes it sound like we're loading multiple modules which confused me at a bit at first but really we're loading the same module once for each device.